ironfish/src/sdk.ts

```diff
@@ -133,9 +145,16 @@ export class IronfishSdk {
     }
 
     let client: RpcSocketClient
+    const rpcAuthToken = internal.get('rpcAuthToken')
```

Comment:
any reason we assign this here instead of just passing it in directly like all of our other config values?

Reply:
hmm because currently the ```init()``` does not have internal pass in